tapdance/conjure.go

```diff
@@ -292,6 +292,8 @@ func DialConjure(ctx context.Context, cjSession *ConjureSession, registrationMet
 
 	cjSession.setV6Support(both)
 
+	// TODO: WebRTC Init Connection and get LocalSDP, send with Register()
```

Comment:
Probably should not merge this whole PR until we have the rest completed; we can merge the protobuf things separately for now